Dixit_Logic/Classes/DixitGame.cs

```diff
@@ -198,7 +220,7 @@ public void EvaluatePoints()
                     
 
                     //This section cheks whether all players has figured out the original card
-                    if (goodGuess != (_actGameState.Players.Count - 1))
+                    if (goodGuess < (_actGameState.Players.Count - 1))
                     {                        
                         _actGameState.Points[_actGameState.ActualPlayer] = _actGameState.Points[_actGameState.ActualPlayer] + 3;
                     }
```

Comment:
isn't it the rule that if everyone figures out the original card then the player gets no point because the hint was too obvious?